validation/src/main/resources/application.conf

```diff
@@ -1,6 +1,7 @@
 akka {
-  stdout-loglevel = "DEBUG"
-  loglevel = "OFF"
+  stdout-loglevel = "OFF"
```

Comment:
Do we want to turn this to `OFF`?  Sounds like we'll lose startup logs if we do this:
* http://doc.akka.io/docs/akka/2.5.3/scala/logging.html#logging-to-stdout-during-startup-and-shutdown